packages/sources/vue/src/components/overlays/VtmnModal/VtmnModal.vue

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
+import '@vtmn/css-modal/dist/index-with-vars.css';
+import { computed, defineComponent, PropType, reactive } from 'vue';
+import { VtmnButton } from '../../index';
+
+export default /*#__PURE__*/ defineComponent({
+  name: 'VtmnAlert',
+  components: { VtmnButton },
+  inheritAttrs: false,
+  props: {
+    show: {
+      type: Boolean as PropType<Boolean>,
+      default: false,
+    },
+    animationDisabled: {
+      type: Boolean as PropType<Boolean>,
+      default: false,
+    },
+  },
+  emits: ['close'],
+  setup(props, { emit }) {
+    props = reactive(props);
+
+    const handleClose = (event: Event) => {
+      return emit('close', (event.target as HTMLInputElement).value);
+    };
+
+    return {
+      classes: computed(() => ({
+        'vtmn-modal': true,
+        show: !props.animationDisabled,
```

Comment:
I don't understand what you are trying to do with the show: !props.animationDisabled.

If it is a class you should add quotes around it. Maybe you want to create an other computed ref ?

Reply:
But @GaspardMathon I do agree with @ThomasLaigneauDecathlon for the fact that's it's weird to link "show" with the prop "animationDisabled", it's not the same purpose.